src/audio/audio.cpp

```diff
@@ -251,6 +259,8 @@ bool Audio::initInput(QString inDevDescr)
         return false;
     }
 
+    privateSetGain(Settings::getInstance().getInVolume());
```

Comment:
Just to tidy up, you need to make sure the input values from the settings are sane (since users with old configurations may contain some very wacky values). I'd suggest defaulting to unity gain (0.0 dB) if values are out of range.